python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -224,6 +224,19 @@ def remove(self):
 
         return
 
+    def get(self, localPath=''):
```

Comment:
Yes. But now we need to introduce some complex mechanism/logic into the manager or a copy command into the baseclass which breaks user scripts and replaces 1 file transfer with multiple ones. 
The ability to transfer a file directly with diracfile or other is a very powerful/useful  feature that is lost by removing this ability. How would you suggest we go about re-adding this?